plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/s3/S3FileSystemExchangeStorage.java

```diff
@@ -755,7 +782,7 @@ public void onFailure(Throwable ignored)
                 {
                     // Rely on caller to abort in case of exceptions during finish
                 }
-                }, directExecutor());
+            }, directExecutor());
```

Comment:
This change can be reverted right ?

Reply:
muscle memory with  option+cmd+L  before commit ;)